rkt/run.go

```diff
@@ -328,6 +340,22 @@ func (pl *portList) Type() string {
 	return "portList"
 }
 
+// flagStringList implements the flag.Value interface to contain a set of strings
+type flagStringList []string
```

Comment:
I don't think we have a good place to put new types used in several, but not all commands. But often `rkt.go` serves as a dumpster for this kind of stuff. ;)